src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2544,3 +2544,248 @@ Random.shuffle!(df::AbstractDataFrame) =
     permute!(df, randperm(nrow(df)))
 Random.shuffle!(r::AbstractRNG, df::AbstractDataFrame) =
     permute!(df, randperm(r, nrow(df)))
+
+"""
+    insertcols(df::AbstractDataFrame[, col], (name=>val)::Pair...;
+               after::Bool=false, makeunique::Bool=false, copycols::Bool=true)
+
+Insert a column into a copy of `df` data frame using the [`insertcols!`](@ref)
+function and return the newly created data frame.
+
+See [`insertcols!`](@ref) for the explanation of arguments and examples.
```

Comment:
Copy the rest of the docstring here? That's more convenient and we do that e.g. for `sort`/`sort!`.

Reply:
OK - done.